pkg/clock/clock.go

```diff
@@ -0,0 +1,29 @@
+package clock
+
+import (
+	"time"
+)
+
+// Clock is an interface to store values ​​of type time
+type Clock interface {
+	Now() time.Time
+}
+
+// Get value from environment variable
```

Comment:
environment?

Reply:
No, this is the defaultBackend which is used to configure the defaultBackend.

Reply:
Ok; good. So it is not related to environment. Good. Closing it.